### PR TITLE
697 fix

### PR DESCRIPTION
**Pull request by domitermonika**
> Pull Request Template
>
> Use this template to quickly create a well written pull request. Delete all quotes before creating the pull request.

## Description

> Please include a summary of the changes and which issue has been fixed. Please also include relevant motivation
> and context. List any dependencies that are required for this change.

Fixes # (issue)

> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- New connector (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce.
> Please also list any relevant details for your test configuration.

- Test A
- Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/WhatsAppComponentHandler.java [hunk 1]
```diff
@@ -37,6 +38,7 @@ public class WhatsAppComponentHandler implements ComponentHandler {
             "WhatsApp is a free-to-use messaging app offering end-to-end encrypted chat, voice, and " +
                 "video communication, along with document and media sharing, available on multiple platforms.")
         .connection(CONNECTION_DEFINITION)
+        .categories(ComponentCategory.COMMUNICATION)
```
Comment (monikakuster) [inline server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/WhatsAppComponentHandler.java:41]: @LukaLjubic za svaki connector mozemo definirati jednu ili vise kategorija kojima pripada. U ComponetCategory enumeraciji mozes vidjeti koje sve imamo

File: server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/connection/WhatsAppConnection.java [hunk 2]
```diff
@@ -16,31 +16,39 @@
...
                     string(SYSTEM_USER_ACCESS_TOKEN)
                         .label("System user access token")
                         .required(true),
                     string(PHONE_NUMBER_ID)
                         .label("Phone number ID")
-                        .required(true)));
+                        .required(true))
+                .apply((connectionParameters, context) -> ofHeaders(
+                    Map.of(AUTHORIZATION,
+                        List.of(BEARER + " " + connectionParameters.getString(SYSTEM_USER_ACCESS_TOKEN))))));
```
Comment (monikakuster) [inline server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/connection/WhatsAppConnection.java:52]: @LukaLjubic sjecas se da smo vec za headere rekli da ih mozemo definirati u connection pomocu apply(). To si vec koristio kod Intercom connectora

File: server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/WhatsAppComponentHandler.java (hunk 1 above)
Comment (LukaLjubic) [reply server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/WhatsAppComponentHandler.java:41]: Oprosti.. nisam vidio. Za druge stavim kategoriju

Comment (monikakuster) [reply server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/WhatsAppComponentHandler.java:41]: Np, od proslog tjedna su uvedene kategorije i dodane svim postojecim. Ali nadalje kako kreiras novi nemoj ih zaboravit dodat :)

File: server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/action/WhatsAppSendMessageAction.java
Comment (monikakuster) [inline server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/action/WhatsAppSendMessageAction.java:1]: @LukaLjubic ovdje nisi definirao outputSchemu jer nisi mogao testirat ili?

Comment (LukaLjubic) [reply server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/action/WhatsAppSendMessageAction.java:1]: Nisam mogao dosta vremena. I onda kad sam napokon uspio jednostavno sam zaboravio.. dodam ja može?

Comment (monikakuster) [reply server/libs/modules/components/whatsapp/src/main/java/com/bytechef/component/whatsapp/action/WhatsAppSendMessageAction.java:1]: Moze, to si stavi na TODO listu NAKON Fine :smile: :smile:

**Pull request merged**